src/Randomizer.SMZ3.Tracking/Configuration/Yaml/BCU/responses.yml

```diff
@@ -3,6 +3,12 @@
   Weight: 0.1
 - Text: My fate is sealed. I have <break strength='weak'/> no choice.
   Weight: 0.5
+- Text: This uh <break time='100ms'/> next band asked me not to read this but goddamn it, I'm going to read it anyway, because I wrote it, and it's the truth. I fucking love this band! They are the best band ever, period! Ladies and gentlemen, Tracker!
+  Weight: 0.05
+- Text: Hey you. You're finally awake. You were trying to cross the border right? Walked right into that Imperial ambush, same as us, and that thief over there. Damn you Stormcloaks.
+  Weight: 0.01
+- Text: Toggled; Oh for the love of; Excuse me one second. <prosody volume='soft'>God dammit this had better be good.</prosody> <voice gender='male'><prosody volume='x-soft'>Mom says it's my turn to do the tracking.</prosody></voice> <prosody volume='soft'>The FUCK it is. Now get out of here you little shit, the seed's already started.</prosody> Sorry, I'm here; just had to deal with my little brother. Let's roll.
```

Comment:
Except for this one. I think Shadow_Hog came up with this one? I don't remember who exactly, but this one's definitely a BCU thing.